README.md

```diff
@@ -3,52 +3,64 @@
 If you've used Hardhat 👷‍♀️👷‍♂️ and want to develop for Starknet <img src="https://starkware.co/wp-content/uploads/2021/07/Group-177.svg" alt="starknet" width="18"/>, this plugin might come in hand. If you've never set up a Hardhat project, check out [this guide](https://hardhat.org/tutorial/creating-a-new-hardhat-project.html).
 
 ## Contents
-- [Install](#install)
```

Comment:
Similar to config.yml, I see formatting has been applied. I've been avoiding that change as it makes the review difficult. If you could revert the formatting changes, then I'd be able to review more easily.